README.md

```diff
@@ -89,18 +89,21 @@ Use the provided snapshots to quickly sync your Juno node with the current state
 | Network | Version | Size | Block | Download Link |
 | ------- | ------- | ---- | ----- | ------------- |
 | mainnet | >=v0.4.0  | 29.4 GB | 100713 | [juno_mainnet_100713.tar](https://juno-snapshot.s3.us-east-2.amazonaws.com/mainnet/juno_mainnet_v0.4.0_100713.tar) |
+| mainnet | **>=v0.5.0**  | **36.5 GB** | **136902** | [**juno_mainnet_136902.tar**](https://juno-snapshot.s3.us-east-2.amazonaws.com/mainnet/juno_mainnet_v0.5.0_136902.tar) |
 
 #### Goerli
 
 | Network | Version | Size | Block | Download Link |
 | ------- | ------- | ---- | ----- | ------------- |
 | goerli | >=v0.4.0 | 31.6 GB | 830385 | [juno_goerli_830385.tar](https://juno-snapshot.s3.us-east-2.amazonaws.com/goerli/juno_goerli_v0.4.0_830385.tar) |
+| goerli | **>=v0.5.0** | **32.3 GB** | **839969** | [**juno_goerli_839969.tar**](https://juno-snapshot.s3.us-east-2.amazonaws.com/goerli/juno_goerli_v0.5.0_839969.tar) |
```

Comment:
Can we run Juno v0.4 with the new snapshots? If not, we should put `v0.4.x` for the old versions (no longer `>=`). Otherwise, we should change `>=v0.5.0` to `>=v0.4.0`.

Reply:
Yeah we can - changed `>=v0.5.0` to `>=v0.4.0`.